membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -90,11 +87,12 @@ class AccountController(
         Left(badRequest("Malformed request. Expected a valid reason for cancellation."))
       }
 
-  def cancelSubscription(subscriptionName: memsub.Subscription.Name): Action[AnyContent] =
+  def cancelSubscription(subscriptionNameString: String): Action[AnyContent] =
```

Comment:
I changed this (and a couple of other methods) because there was a method further down the file that just called this method with a wrapper on the subscription name.